packages/appstore/src/components/containers/listing-container.tsx

```diff
@@ -14,9 +14,7 @@ type TListingContainerProps = {
 type TOptionsProps = Pick<TListingContainerProps, 'title' | 'description' | 'is_deriv_platform'>;
 type TSwitcherProps = Pick<TListingContainerProps, 'is_deriv_platform'>;
 
-const Options = ({ title, description, is_deriv_platform }: TOptionsProps) => {
-    if (!is_deriv_platform) return null;
-
+const Options = ({ title, description }: TOptionsProps) => {
```

Comment:
Removed the is_deriv_platform as we are using the component in both CFD and option page